releasenotes/notes/remove-backend-props-unitary-synth-0dca60b56ccd9ee3.yaml

```
@@ -0,0 +1,11 @@
+---
+upgrade_transpiler:
+  - |
+    The ``backend_properties`` input argument has been removed from the following objects:
+    
+    * :class:`.UnitarySynthesis` transpiler pass
+    * :func:`.generate_translation_passmanager` function
+
+    This argument required a ``BackendProperties`` object, deprecated since qiskit 1.2, as it
+    belonged to the deprecated BackendV1 workflow. Constraints such as gate error and durations 
```

Do you want to mention that `BackendProperties` is removed in this release?
```suggestion
    This argument required a ``BackendProperties`` object, deprecated since Qiskit 1.2, as it
    belonged to the deprecated ``BackendV1`` workflow. Constraints such as gate error and durations 
```

---

I wrote the reno before #13722, which completes the removal... it is a bit weird to have UnitarySynthesis singled out, so I am thinking... should we remove the reno alltogether?

---

I decided to remove the reno, as the changes are already reflected in the reno of https://github.com/Qiskit/qiskit/pull/13722. I will change the label to Changelog:None to make sure there are no issues with the release bot.

---

Agreed, it should all be in a single reno to understand what's going on :+1: